web/src/message_fetch.ts

```diff
@@ -153,16 +153,17 @@ function process_result(data: MessageFetchResponse, opts: MessageFetchOptions):
     // messages not tracked in unread.ts during this fetching process.
     message_util.do_unread_count_updates(messages, true);
 
+    const ignore_found_newest = true;
```

Comment:
Why is this called `ignore_found_newest`? I'm not quite sure what the name should be.

Reply:
I feel like we want something that's more direct about the meaning, maybe `can_safely_extend_message_range`?

Reply:
I just wanted this parameter to be explicitly saying what it does whenever it is being used.

Reply:
Yeah, I think it's usually better to communicate what information the caller knows, at least in cases like this where "ignoring found_newest" probably won't have a clear meaning to anyone reading the caller.

I'm going to merge, but maybe you can rename this in a follow-up.

Reply:
Opened #32765